src/main/cpp/chassis/swerve/headingStates/FaceGoalHeading.cpp

```diff
@@ -15,30 +15,30 @@
 
 // Team302 Includes
 #include <chassis/swerve/headingStates/FaceGoalHeading.h>
+#include <utils/logging/Logger.h>
```

Comment:
This class either has been deleted or will be deleted

Reply:
Fixed with merge